__tests__/app.test.js

```diff
@@ -353,25 +353,38 @@ describe("GET /api/articles (topic query)", () => {
         });
       });
   });
+});
+test("400: returns error when non existant topic is given", () => {
+  return request(app)
+    .get("/api/articles?topic=elephants")
+    .expect(400)
+    .then(({ body }) => {
+      const { msg } = body;
+      expect(msg).toBe("bad request");
+    });
+});
 
-  test("400: returns error when non existant topic is given", () => {
-    return request(app)
-      .get("/api/articles?topic=elephants")
-      .expect(400)
-      .then(({ body }) => {
-        const { msg } = body;
-        expect(msg).toBe("bad request");
-      });
-  });
+test("200: returns empty array when the topic exists but there are no articles with that topic.", () => {
+  return request(app)
+    .get("/api/articles?topic=paper")
+    .expect(200)
+    .then(({ body }) => {
+      const { articles } = body;
+      expect(articles).toHaveLength(0);
+    });
+});
 
-  test("200: returns empty array when the topic exists but there are no articles with that topic.", () => {
+describe("GET /api/articles/:article_id (comment_count)", () => {
+  test("200: responds with object with a count of the total of the articles with the requested article_id", () => {
     return request(app)
-      .get("/api/articles?topic=paper")
+      .get("/api/articles/1")
       .expect(200)
       .then(({ body }) => {
-      const {articles} = body
-      expect(articles).toHaveLength(0);
+        const { article } = body;
+        expect(article[0]).toMatchObject({
+          article_id: 1,
+          comment_count: "11",
+        });
```

Comment:
Nice work checking that the new property has been added to the get articles by ID endpoint. You could also check that all the other values for the article are returned here too.